models.tsx

```diff
@@ -0,0 +1,8 @@
+export interface Vehicle {
```

Comment:
W TS tego typu pliki nazywa się raczej `types.ts` a nie `models.ts` -> To nie są do końca modele w rozumieniu innych frameworków. To są tylko typy jakimi operujemy.